src/overview/index.ts

```diff
@@ -45,32 +46,42 @@ export async function overviewCmdHandler(context: vscode.ExtensionContext, opera
 
   context.globalState.update(KEY_OVERVIEW_LAST_SHOW_TIME, Date.now().toString());
 
-  overviewView.iconPath = vscode.Uri.file(path.join(context.extensionPath, 'logo.lowres.png'));
-  let buffer = await readFile(path.join(context.extensionPath, './out/assets/overview/index.html'));
-  overviewView.webview.html = buffer.toString();
+  await initializeOverviewView(context, overviewView, onDidDisposeWebviewPanel);
+}
 
-  overviewView.onDidDispose(() => {
-    overviewView = undefined;
-  });
+function onDidDisposeWebviewPanel() {
+  overviewView = undefined;
+}
+
+async function initializeOverviewView(context: vscode.ExtensionContext, webviewPanel: vscode.WebviewPanel, onDisposeCallback: () => void) {
+  webviewPanel.iconPath = vscode.Uri.file(path.join(context.extensionPath, 'logo.lowres.png'));
+  webviewPanel.webview.html = await loadHtmlContent(context);
+
+  webviewPanel.onDidDispose(onDisposeCallback);
```

Comment:
Oh, sorry I missed this part. Since the Listener `onDidDispose()` is a `Disposable` as well, pushing into the subscription is needed.

You can take a reference here: https://github.com/jdneo/vscode-leetcode/blob/master/src/leetCodeResultProvider.ts#L22-L24

Reply:
I actually found more. Thanks for the reminder!